src/gwc/src/test/java/org/geoserver/gwc/GWCIntegrationTest.java

```diff
@@ -159,6 +159,7 @@ public void cleanupDispatcherRequest() {
 
     private void prepareDataDirectory(SystemTestData testData) throws Exception {
         Catalog catalog = getCatalog();
+        catalog.setDefaultWorkspace(getCatalog().getWorkspaceByName(BASIC_POLYGONS.getPrefix()));
```

Comment:
No need to make this default in a test actually using per workspace services.

Reply:
understood, will do